webpack/assets/javascripts/react_app/components/HostsIndex/index.js

```diff
@@ -175,6 +188,14 @@ const HostsIndex = () => {
     >
       {__('Delete')}
     </DropdownItem>,
+    <DropdownItem
+      ouiaId="build-hosts-dropdown-item"
+      key="build-hosts-dropdown-item"
+      onClick={setModalOpen}
+      isDisabled={selectedCount === 0}
+    >
+      {__('Build Hosts')}
```

Comment:
As per the mockup:

```suggestion
      {__('Build management')}
```

(and don't forget sentence case, not title case)